examples/odata-cli/.eslintrc

```
@@ -1,5 +1,6 @@
 {
     "extends": ["../../.eslintrc"],
+    "parserOptions": { "project": "./tsconfig.esbuild.json" },
```

```suggestion
    "parserOptions": { "project": "./tsconfig.eslint.json" },
```
As discussed please rename the config file

---

Thanks, @devinea for catching this typo. I renamed the files.